pkg/kubelet/server/server.go

```diff
@@ -878,6 +891,63 @@ func (s *Server) getPortForward(request *restful.Request, response *restful.Resp
 	proxyStream(response.ResponseWriter, request.Request, url)
 }
 
+// checkpoint handles the checkpoint API request. It checks if the requested
+// podNamespace, pod and container actually exist and only then calls out
+// to the runtime to actually checkpoint the container.
+func (s *Server) checkpoint(request *restful.Request, response *restful.Response) {
+	if request == nil || response == nil {
```

Comment:
Is this check needed? I don't see it in the other handlers.

Reply:
Just wanted to be on the safe side. Removed the check now.